### ISSUE TITLE
feature request : sigma designs color 400

### ISSUE THREAD

**Issue by wimmatthijs** [viti95/V95Bench#1]
i have this card but the video modes are special.
i want to make sure it is working correcty.
Could it be possible to add it to the test?
http://www.seasip.info/VintagePC/sigma400.html

**Comment by viti95** [comment]
Yeah for sure, I personally don't have this video card, but I can add support to it and test it with 86Box. I'll add an option to test patterns with different video cards.

**Comment by wimmatthijs** [comment]
Wow, great news! I have this card but the software support is quite rare.
I can run the test on hardware to see if they act the same.
I have the Version One (long ISA) version of this card with bios version 2.49.
I will also set up the environment in parallel to understand more about this videocard and its capabilites.
Thanks again.

**Comment by wimmatthijs** [comment]
i was trying to set up the open watcom v2 ide but i failed to do so on my mabcook air M1, even tried a WIN98 emulator but it doesnt want to work.
**update:** i was able to compile the code in the virtual machine myself now, going to move forward by trying to write the RAM planes

**Comment by viti95** [comment]
I do use Linux (and sometimes Windows) to cross build my DOS projects, it's much faster compiling compared to the DOS OpenWatcom. Haven't tried on Apple based devices, since I don't have one right now.
